app/javascript/stylesheets/application.scss

```diff
@@ -1,6 +1,25 @@
 @import "~bootstrap/scss/bootstrap";
 @import "./_sidebar";
+@import url('https://fonts.googleapis.com/css2?family=Lobster&display=swap');
 
 #page-content-wrapper {
   width: 100%;
 }
+
+.big-button {
```

Comment:
I think you can use the Bootstrap pseudo-class 'w-100' to set the width to 100%. So you don't need to define this here.

Reply:
Done